pkgs/development/libraries/re2/default.nix

```diff
@@ -21,27 +20,29 @@ stdenv.mkDerivation rec {
     owner = "google";
     repo = "re2";
     rev = version;
-    sha256 = "sha256-UontAjOXpnPcOgoFHjf+1WSbCR7h58/U7nn4meT200Y=";
+    hash = "sha256-UontAjOXpnPcOgoFHjf+1WSbCR7h58/U7nn4meT200Y=";
   };
 
-  preConfigure = ''
```

Comment:
Modifying this package should be done in a seperate commit, especially because it seems to be triggering rebuilds

Reply:
Yes, it's actually done here #196123.